HelpSource/Tutorials/JITLib/proxyspace_examples.schelp

```diff
@@ -575,7 +575,7 @@ SynthDef("grain", { arg i_out = 0, pan;
 	var env;
 	env = EnvGen.kr(Env.perc(0.001, 0.003, 0.2),doneAction: Done.freeSelf);
 	Out.ar(i_out, Pan2.ar(FSinOsc.ar(Rand(1000,10000)), pan) * env)
-}).send(s);
+}).add;
```

Comment:
keep send, it was used intentionally for most efficient way.

Reply:
Thank you, and my apologies for the change. I have reverted it in https://github.com/supercollider/supercollider/pull/7117/commits/5b60bd3d0307e40ba1d3133d4800dc5030070cfb.